approvaltests/pytest/pytest_plugin.py

```diff
@@ -0,0 +1,4 @@
+from .pytest_config import set_pytest_config
+
+def pytest_configure(config):
```

Comment:
I wasnt sure how this was all wired together - what calls pytest_configure?

Reply:
This is a pytest plugin, I think pytest treats any function starting with `pytest_` in a plugin module as an entry point to the plugin and sends the config as a fixture.